Lib9c/Action/SellCancellation.cs

```diff
@@ -59,13 +59,18 @@ public Result(BxDictionary serialized) : base(serialized)
             [ProductIdKey] = orderId.Serialize(),
             [SellerAvatarAddressKey] = sellerAvatarAddress.Serialize(),
             [ItemSubTypeKey] = itemSubType.Serialize(),
+            [TradableIdKey] = tradableId.Serialize(),
         }.ToImmutableDictionary();
 
         protected override void LoadPlainValueInternal(IImmutableDictionary<string, IValue> plainValue)
         {
             orderId = plainValue[ProductIdKey].ToGuid();
```

Comment:
I think we should change `ProductIdKey` to `OrderIdKey`

Reply:
I agree. However, if we change the key, the internal chain needs to be reset, so I left this.